projects/admin/src/app/record/formly/type/entity-typeahead/entity-typeahead.component.ts

```diff
@@ -165,11 +180,16 @@ export class EntityTypeaheadComponent extends FieldType implements OnInit {
       const newSectionSuggestions = [];
       let lastGroup = '';
       section.forEach(suggestion => {
-        if (suggestion?.group && suggestion.group !== lastGroup){
+        if (suggestion?.group && suggestion.group !== lastGroup) {
           lastGroup = suggestion.group;
           newSectionSuggestions.push(new TypeaheadMatch(suggestion.group, suggestion.group, true));
         }
-        newSectionSuggestions.push(new TypeaheadMatch(suggestion, suggestion.label, false));
+        // We create a section header only if the value is present.
+        // If we have a suggestion with no value, it's because it's been added manually (see above),
+        // no need to create a new suggestion.
```

Comment:
Section header will be created only if suggestion defines a valid value.
If the suggestion doesn't define any value, this is because this is a fake suggestion ; added manually (see above).
In such case, don't create any `TypeaheadMatch`.